core/power/power_table_error.hpp

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef FILECOIN_CORE_POWER_POWER_TABLE_ERROR_HPP
+#define FILECOIN_CORE_POWER_POWER_TABLE_ERROR_HPP
+
+#include "common/outcome.hpp"
+
+namespace fc::power {
+
+  /**
+   * @brief Type of errors returned by Power Table
+   */
+  enum class PowerTableError {
+    NO_SUCH_MINER = 1,
+    NEGATIVE_POWER,
+
+    UNKNOWN
```

Comment:
And what is this for? It's not quite clear to me in what case will you intentionally return this value.

Reply:
Removed